pkg/skaffold/build/buildpacks/logger.go

```diff
@@ -30,12 +33,27 @@ type logger struct {
 }
 
 func NewLogger(out io.Writer) logging.Logger {
+	// If out is not a terminal, let's make sure no colors are printed.
+	if _, isTerm := util.IsTerminal(out); !isTerm {
+		out = colorable.NewNonColorable(out)
+	}
+
+	l := logrus.New()
+	l.SetOutput(out)
+	l.SetFormatter(new(plainFormatter))
```

Comment:
I'd appreciate a comment here as to why this formatter is required

Reply:
Added